core-toolbar.css

```diff
@@ -87,20 +111,24 @@ polyfill-next-selector { content: ':host.core-narrow .toolbar-tools, .core-narro
   left: 0;
 }
 
+/* ******************************* */
+/* TODO: Should content be styled? */
```

Comment:
Should it? My guess is that if this is _not_ a paper element, the answer is, "no." Can @frankiefu or @morethanreal chime in on this?